test/test_hashable.pf

```diff
@@ -0,0 +1,29 @@
+@test
+subroutine test_hashable_int()
```

Comment:
same comment here.

Reply:
Updated!

Reply:
Sorry, just updated.  Missed this file in my previous commit.